src/main/java/me/zeroeightsix/fiber/exception/RuntimeFiberException.java

```diff
@@ -1,5 +1,9 @@
 package me.zeroeightsix.fiber.exception;
 
+/**
+ * An exception thrown by fiber when something goes unrecoverably wrong.
```

Comment:
Unrecoverably wrong but does not have to be caught?

Reply:
Yes, because `RuntimeFiberException` is something only thrown when you make very obvious mistakes: it's an exception thrown when e.g. you put a string constraint on a numerical setting.
It's when the developer does something clearly wrong, but in a method that would be incredibly annoying to have to try/catch each time you use it.

Reply:
I'd say some info like that sentence 'thrown when e.g. you put a string constraint on a numerical setting.' would be enough to make that clear.